src/models/split_transfers.py

```diff
@@ -99,32 +103,38 @@ def _process_token_transfers(
                     # Reinsert since there is still an amount owed.
                     self.overdrafts[solver] = overdraft
                     continue
-            if solver in cow_redirects:
-                # Redirect COW rewards to reward target specific by VouchRegistry
-                redirect_address = cow_redirects[solver].reward_target
-                log_saver.print(
-                    f"Redirecting solver {solver} COW tokens "
-                    f"({transfer.amount}) to {redirect_address}",
-                    category=Category.REDIRECT,
-                )
-                transfer.receiver = redirect_address
+            transfer.redirect_from(cow_redirects, log_saver)
```

Comment:
> Does the _process_rewards really need the overdraft if conditions?

@josojo - you appear to be referring to this

and in response I would say that they do not and this is why they have been removed here. This logic is now fully encapsulated in `transfer.redirect_from`